@xen-orchestra/web-core/lib/components/ui/alarm-item/UiAlarmItem.vue

```diff
@@ -0,0 +1,144 @@
+<!-- v1 -->
+<template>
+  <li class="ui-alarm-item" :class="className">
+    <div class="content">
+      <div class="title-progress text-ellipsis">
+        <UiButtonIcon
+          v-if="alarm.description"
+          :icon="isDescriptionVisible ? faAngleDown : faAngleRight"
+          size="small"
+          accent="brand"
+          :target-scale="2"
+          @click="toggleDescription()"
+        />
+        <span v-tooltip="alarm.label" class="typo-body-regular text-ellipsis">
```

Comment:
```suggestion
        <span v-tooltip class="typo-body-regular text-ellipsis">
```

Check if it's work but normally you do not need to add this